doc/source/examples/dem/granuheap/granuheap.rst

```diff
@@ -0,0 +1,259 @@
+
+
+----------------------------
+Files Used in This Example
+----------------------------
+
+All files mentioned below are located in the example's folder (``examples/dem/3d-granuheap``).
+
+- Mesh file: ``cylinder.msh`` and ``support.msh``
+- Parameters file for the problem: ``granuheap.prm``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the flow of water-wet sand during a Granuheap experiment. First, the packing of particles is generated by inserting particles inside a cylinder. After 1.6s, the cylinder rises, and the particles can flow freely, producing a heap. We compare the numerical results with our experimental results. To remain consistent with granuheap, we use the same metrics in our simulations: a cylinder of 0.01m in diameter. For more information about granuheap procedures, visit the `granuheap web page <https://www.granutools.com/en/granuheap>`_. To match our experimental results, we test multiple combinations of parameters in our simulations. We use the work of Hu *et al.* `[1] <https://doi.org/10.1016/j.powtec.2021.11.049>`_ as a basis for the choice of variable parameters.
+
+
+--------------
+Parameter File
+--------------
+
+Mesh
+~~~~~
+
+The domain we simulate is a rectangular box measuring :math:`0.045\times0.02\times0.02` metres, made using the deal.ii grid generator.  The grid is refined 6 times using the ``set initial refinement`` parameter.
+
+.. code-block:: text
+
+  subsection mesh
+    set type               = dealii
+    set grid type          = subdivided_hyper_rectangle
+    set grid arguments     = 3,1,1 : -0.010,-0.01,-0.01 : 0.035, 0.01,0.01 : true
+    set initial refinement = 6
+  end
+  
+  
+Boundaries Conditions
+~~~~~~~~~~~~~~~~~~~~~
+
+The bottom of the domain is set as an outlet to reduce the time of the simulation. Thus, all particles leaving the support plate will no longer be part of the simulation. 
+
+.. code-block:: text
+  
+  subsection DEM boundary conditions
+    set number of boundary conditions = 1
+
+    subsection boundary condition 0
+      set boundary id         = 0
+      set type                = outlet
+    end
+  end
+
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The particles are polydispersed with a density of 1922 :math:`kg/m^3`. 400 000 particles would be needed to match the 2 g of sand used in the experiment. However, to reduce the simulation's duration of this example, we insert only 200 000 particles.
+
+According to Hu *et al.* `[1] <https://doi.org/10.1016/j.powtec.2021.11.049>`_, the rolling friction and the surface energy parameters impact the final simulated angle of repose the most. Thus, we simulate twelve combinations of those parameters. The ``rolling friction`` is between 0.3 and 0.7, and the ``surface energy`` is between 0.0010 and 0.0100. The ``granuheap.prm`` file of this example is specifically for a ``rolling friction`` of 0.5 and a ``surface energy`` of 0.0100.
```

Comment:
I would look at Audrey's pneumatic conveying example she had found a nice way to cite articles.